code/RegistryPage.php

```diff
@@ -355,6 +355,12 @@ public function export($request)
 
     public function show($request)
     {
+        // if Id is empty or it is not numeric, then return an error page
+        if(empty($request->param('ID')) || !is_numeric($request->param('ID')))
```

Comment:
I don't think this will work for PHP 5.3. Maybe just `!is_numeric($request->param('ID')` would be fine. We don't need to check whether it's set, because `SS_HTTPResponse::param` will return null if it's not.

Would you also mind formatting this for PSR-2 please?